lib/helpers.js

```diff
@@ -10,7 +10,16 @@ export const {
     CONTEXT_URL: CREDENTIALS_CONTEXT_V2_URL
   }
 } = credentialsContextV2;
+// Z and T can be lowercase
+// RFC3339 regex
+export const dateRegex = new RegExp('^(\\d{4})-(0[1-9]|1[0-2])-' +
```

Comment:
The year validation in that RegExp is more strict than mine and it also handles years larger than 4 numbers so that seems like an advantage. it also has a different validator for T or Z so yeah I'll adapt it to this lib today.

Reply:
Addressed here: https://github.com/digitalbazaar/vc/pull/158/commits/80d30d7163bfc2abc950a510c4e120ff67ecf900